src/components/Clock.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import propTypes from 'prop-types';
+
+export class Clock extends React.Component {
+  state = {
+    time: (new Date()).toLocaleTimeString(),
+  };
+
+  componentDidMount() {
+    this.setState({
+      interval: setInterval(() => {
+        this.setState({ time: (new Date()).toLocaleTimeString() });
+        // eslint-disable-next-line
+        console.log(this.state.time);
+      }, 1000),
```

Comment:
```suggestion
    this.interval = setInterval(() => {
        this.setState({ time: (new Date()).toLocaleTimeString() });
        // eslint-disable-next-line
        console.log(this.state.time);
      }, 1000),
```

No need to write setInterval to the state